test/test_index_string.cpp

```diff
@@ -773,8 +773,8 @@ TEST(StringIndex_FindAllNoCopyCommonPrefixStrings)
         const IntegerColumn results_c(Allocator::get_default(), ref_type(results.payload));
         CHECK_EQUAL(results_c.get(results.start_ndx), start_row + 1);
         CHECK_EQUAL(results_c.get(results.start_ndx + 1), start_row + 2);
-        CHECK_EQUAL(col.get(results_c.get(results.start_ndx)), spc);
-        CHECK_EQUAL(col.get(results_c.get(results.start_ndx + 1)), spc);
+        CHECK_EQUAL(col.get(size_t(results_c.get(results.start_ndx))), spc);
```

Comment:
I don't think we should use such core internals in unit tests, as I see the unit tests as being an "app" that uses mostly just public API (such that if we change these internals, we don't need to change the tests).

If there row-number exceeds 32 bits in this test, it will be caught by the CHECK_EQUAL